code/datums/actions/items/cult_dagger.dm

```diff
@@ -18,7 +18,7 @@
 
 /datum/action/item_action/cult_dagger/Trigger(trigger_flags)
 	for(var/obj/item/held_item as anything in owner.held_items) // In case we were already holding a dagger
-		if(istype(held_item, /obj/item/melee/cultblade/dagger))
+		if(istype(held_item, /obj/item/melee))
```

Comment:
Seems like it would be very easy to false-positive-trigger this if it is scoped to all melee items. I mean that even includes sec batons.

Reply:
stupid old useless shitcode

Reply:
One way to fix it is to make this just trigger attack self of `target` if `target` in `held_items` I guess